packages/client/lib/sync/txpool.ts

```diff
@@ -12,17 +12,23 @@ export interface TxPoolOptions {
 
 type TxPoolObject = {
   tx: TypedTransaction
-  hash: string // plain strings without hex prefix
+  hash: UnprefixedHash
   added: number
 }
 
 type HandledObject = {
-  address: string // plain strings without hex prefix
+  address: UnprefixedAddress
+  added: number
+}
+
+type SentObject = {
+  hash: UnprefixedHash
   added: number
 }
 
 type UnprefixedAddress = string
```

Comment:
Just taking a look at how this is used (`UnprefixedAddress`), actually when you do `getSenderAddress().toString()` that will return the 0x-prefixed address (if you look at `Address` class `toString` method).

Two ways to get the unprefixed hash would be:
`getSenderAddress().toString().slice(2)`
or
`getSenderAddress().buf.toString('hex')`

Reply:
Couldn't we also just use `stripPrefix` from `ethereumjs-util`?  I know it's a simple convenience function but is that fruitful in this instance?